locals.tf

```diff
@@ -0,0 +1,4 @@
+locals {
+  default_tags = {}
```

Comment:
are we going to use default tags?
Maybe we can have at least the controller_by( or created_by) tg with value terraform:
  default_tags = {created_by: "terraform"}